docs/sources/operators-guide/deploy-grafana-mimir/upgrade-helm-chart-2.1-to-3.0/_index.md

```diff
@@ -62,6 +62,8 @@ Several parameters that were available in version 2.1 of the mimir-distributed H
 
 1. Decide whether or not you need to update the memcached configuration, which has changed:
 
+   > **Note:** Some Helm values related to memcached have changed. For information about how they affect your Mimir configuration, see step 6 later.
```

Comment:
This is a future documentation bug waiting to happen because you have hard-coded a step number into plain text.